static/app/views/performance/trends/changeExplorer.spec.tsx

```diff
@@ -0,0 +1,301 @@
+import React from 'react';
+import moment from 'moment';
+
+import {initializeData} from 'sentry-test/performance/initializePerformanceData';
+import {act, render, screen, waitFor} from 'sentry-test/reactTestingLibrary';
+
+import ProjectsStore from 'sentry/stores/projectsStore';
+import {PerformanceChangeExplorer} from 'sentry/views/performance/trends/changeExplorer';
+import {
+  COLUMNS,
+  MetricsTable,
+  renderBodyCell,
+} from 'sentry/views/performance/trends/changeExplorerUtils/metricsTable';
+import {
+  NormalizedTrendsTransaction,
+  TrendChangeType,
+  TrendFunctionField,
+} from 'sentry/views/performance/trends/types';
+import {TRENDS_PARAMETERS} from 'sentry/views/performance/trends/utils';
+
+jest.mock(
+  'sentry/utils/getDynamicComponent',
+  () =>
+    ({fixed}) =>
+      fixed
+);
```

Comment:
What does this do?

Reply:
literally no clue, i don't even remember putting this in. Possibly got added in when I committed??